combat-trainer.lic

```diff
@@ -392,8 +392,25 @@ class LootProcess
   end
 
   def should_perform_ritual(prepare_consume)
-    !prepare_consume &&
-      ((@necro_train_first_aid && DRSkill.getxp('First Aid') < 32) || DRSkill.getxp('Thanatology') < 32)
+    return false unless @ritual_type
+    return false if prepare_consume
+    return true if @ritual_type == 'dissect' && DRSkill.getxp('First Aid') < 32
+    return true if @ritual_type == 'harvest' && DRSkill.getxp('Skinning') < 32
+    return true if DRSkill.getxp('Thanatology') < 32
+    true
```

Comment:
Should this be `false`? If it is `true`, then you don't need the `return true if` statements above.

Reply:
yes! thanks